pyrasterframes/src/main/python/pyrasterframes/__init__.py

```diff
@@ -72,21 +72,25 @@ def _convert_df(df: DataFrame, sp_key=None, metadata=None) -> RasterFrameLayer:
 def _raster_join(df: DataFrame, other: DataFrame,
                  left_extent=None, left_crs=None,
                  right_extent=None, right_crs=None,
-                 join_exprs=None) -> DataFrame:
+                 join_exprs=None, resampling_method='nearest_neighbor') -> DataFrame:
     ctx = SparkContext._active_spark_context._rf_context
+    assert resampling_method in ['nearest_neighbor', 'bilinear', 'cubic_convolution', 'cubic_spline', 'lanczos',
+                                 'average', 'mode', 'median', 'max', 'min', 'sum']
```

Comment:
🙇

Reply:
Actually, do you need to do the `lower().strip('_')` thing here?

Reply:
would help